Author: Rolf Leggewie <foss@rolf.leggewie.biz>

improve description for scanbd in init file and add call to status

Index: scanbd/integration/scanbd.debian
===================================================================
--- scanbd.orig/integration/scanbd.debian	2014-10-16 12:25:25.361209803 +0800
+++ scanbd/integration/scanbd.debian	2014-10-16 12:27:32.733841407 +0800
@@ -26,9 +26,8 @@
 # Required-Stop:     $local_fs $remote_fs
 # Default-Start:     2 3 4 5
 # Default-Stop:      0 1 6
-# Short-Description: scanbd start script
-# Description:       This file should be used to construct scripts to be
-#                    placed in /etc/init.d.
+# Short-Description: Scanner button events daemon
+# Description:       scanbd associates actions to scanner buttons
 ### END INIT INFO
 
 # Do NOT "set -e"
@@ -160,9 +159,11 @@
 		;;
 	esac
 	;;
+  status)
+	status_of_proc -p "$PIDFILE" "$DAEMON" "$NAME" && exit 0 || exit $?
+	;;
   *)
-	#echo "Usage: $SCRIPTNAME {start|stop|restart|reload|force-reload}" >&2
-	echo "Usage: $SCRIPTNAME {start|stop|restart|force-reload}" >&2
+	echo "Usage: $SCRIPTNAME {start|stop|restart|force-reload|status}" >&2
 	exit 3
 	;;
 esac
